Tests/Integration/tests/MessengerTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Presta\SitemapBundle\Tests\Integration\Tests;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Messenger\EventListener\StopWorkerOnMessageLimitListener;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Transport\InMemoryTransport;
+use Symfony\Component\Messenger\Worker;
+
+class MessengerTest extends SitemapTestCase
+{
+    private const PUBLIC_DIR = __DIR__ . '/../public';
+    private const GET = Request::METHOD_GET;
+
+    protected function setUp(): void
+    {
+        if (!class_exists(MessageBusInterface::class)) {
```

Comment:
Same thing here : `class_exists` used on an interface instead of `interface_exists`